### ISSUE TITLE
Add 'history' linux alike command in truffle develop [ENHANCEMENT]

### ISSUE THREAD

**Issue by thecircleoflifefree** [trufflesuite/truffle#2865]
Enhancement Request (Nice to have)

Just like in linux where you can run 'history' and see the list of commands that you have recently used.

**Comment by gnidan** [comment]
Yeah! Totally! Good idea!

(How do we do it?)

**Comment by haltman-at** [comment]
Obvious note, but, whenever this happens it should apply to Truffle Console as well.

**Comment by rkalis** [comment]
It looks like the history is accessible from the node repl. I created a very simple test file to check it out, and it appears that you can just access `repl.history` and get a list of all repl history. However, I don't think printing it is included in a builtin command.

Test file:
```js
const repl = require('repl')

const activeRepl = repl.start('> ')
activeRepl.on('exit', () => {
  console.log(activeRepl.history)
})
```

Output:
```js
> 2 + 2
4
> 4 + 0
4
> .exit
[ '.exit', '4 + 0', '2 + 2' ]
```

It also looks like the [repl.history](https://www.npmjs.com/package/repl.history) package can be used to persist this history across executions of `truffle console`. Do we want persistent history, or in-memory?

It makes sense to add a `.history` command to the repl that prints x lines of history (perhaps with a cli parameter for the number of lines?).

If you agree on implementing it like this, I could look into integrating that into the existing `truffle console` and `truffle develop` commands.

**Comment by stvtortora** [comment]
Hi, is anyone working on this? I'd be up to taking a stab at it.

**Comment by haltman-at** [comment]
No, I don't believe any of us are working on this at the moment.
